packages/common/src/lib/format-questionnaire-details.js

```diff
@@ -0,0 +1,199 @@
+			return 'Extractive industry';
+		case 'food-industry':
+			return 'Food industry';
+		case 'infrastructure-projects':
+			return 'Infrastructure projects';
+		case 'mineral-industry':
+			return 'Mineral industry';
+		case 'other-projects':
+			return 'Other projects';
+		case 'production-processing-of-metals':
+			return 'Production and processing of metals';
+		case 'rubber-industry':
+			return 'Rubber industry';
+		case 'textile-industries':
+			return 'Textile, leather, wood and paper industries';
+		case 'tourism-leisure':
+			return 'Tourism and leisure';
+		default:
+			return 'None';
+	}
+};
+
+// TODO the associated question is checkbox meaning there could be multiple answers
+// notificationMethod is currently a string
+// ? should this be an array of strings in schema or need a relation table
+// function will need refactoring when data model corrected
+// /**
+//  * @param {AppealCaseWithAppellant} caseData
+//  */
+// exports.formatNotificationMethod = (caseData) => {
+// 	/**
+// 	 * @type {string[]}
+// 	 */
+// 	const notifcationTypes = [];
+// 	if (caseData.notificationMethod) {
+// 		const types = caseData.notificationMethod.split(' ');
+// 		types.forEach((type) => {
+// 			if (type === 'site-notice') {
+// 				notifcationTypes.push('A site notice');
+// 			}
+// 			if (type === 'letters-or-emails') {
+// 				notifcationTypes.push('Letter sent to neighbours');
+// 			}
+// 			if (type === 'advert') {
+// 				notifcationTypes.push('Press advert');
+// 			}
+// 		});
+// 	}
+// 	if (!notifcationTypes) {
+// 		return 'None';
+// 	}
+// 	return notifcationTypes.join('<br>');
+// };
+
+/**
+ * @param {String | undefined} dateStr
+ */
+exports.formatDate = (dateStr) => {
+	if (!dateStr) {
+		return '';
+	}
+
+	const date = new Date(dateStr);
+
+	const day = date.getDate().toString().padStart(2, '0');
+	const month = (date.getMonth() + 1).toString().padStart(2, '0');
+	const year = date.getFullYear();
+
+	return `${day}-${month}-${year}`;
+};
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatSiteSafetyRisks = (caseData) => {
+	if (caseData.lpaSiteSafetyRisks) {
+		return `Yes<br>${caseData.lpaSiteSafetyRiskDetails}`;
+	} else {
+		return 'No';
+	}
+};
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatProcedurePreference = (caseData) => {
+	if (caseData.lpaProcedurePreference === 'written-representations') {
+		return `Written representations`;
+	} else if (caseData.lpaProcedurePreference === 'hearing') {
+		return `Hearing<br>${caseData.lpaPreferHearingDetails}`;
+	} else if (caseData.lpaProcedurePreference === 'inquiry') {
+		return `Inquiry<br>${caseData.lpaPreferInquiryDetails}<br>Expected duration: ${caseData.lpaPreferInquiryDuration} days`;
+	} else {
+		return '';
+	}
+};
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ */
+exports.formatConditions = (caseData) =>
+	caseData.newConditions === true ? `Yes<br>${caseData.newConditionDetails}` : 'No';
```

Comment:
you can just say `caseData.newConditions` no need to compare it to `true`

Reply:
This is also expressible as 
`caseData.newConditions && `Yes<br>${caseData.newConditionDetails}` || 'No'`